examples/legacy_local/201_customer_keyspace.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2019 The Vitess Authors.
```

Comment:
If there is no change to the content of the file, the copyright year should not change.
If there is significant change, people typically add the new year so that it becomes 2019,2020. I don't think we need to bother with that.

Reply:
Git has some odd logic to distinguish between a rename and a new file. Logically this file is "new" and the old file was moved to examples/legacy_local, but git picked it the other way.

There is a workaround where Rohit could do this in two PRs - first duplicate the examples/local directory, and then second would be to do the edits for the new workflow.

Reply:
Am closing this PR per Morgan's suggestion into two PRs one with a copy into a legacy directory and then modifying current one for the new workflow.